### PR TITLE
Updated PrivateIPAddress var

### PR DESCRIPTION
**Pull request by pselugar**
- Private IpAddress error under Reset-Nic was because of the breaking change from CLI team. [Breaking Change](https://github.com/Azure/azure-cli/pull/25498)
- Updated the field `privateIpAddres` to `privateIPAddress`.
- Re run Reset-Nic for TestVM and validate the change.

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vm repair

**Review by haagha — CHANGES_REQUESTED** [review]
Can you also add to history https://github.com/pselugar/azure-cli-extensions/blob/pselugar-reset-nic-privateip/src/vm-repair/HISTORY.rst

**Review by haagha — APPROVED** [review]

File: src/vm-repair/azext_vm_repair/custom.py [hunk 1]
```diff
@@ -580,7 +580,7 @@ def reset_nic(cmd, vm_name, resource_group_name, yes=False):
         vnet_name = subnet_id_tokens[-3]
         vnet_resource_group = subnet_id_tokens[-7]
         ipconfig_name = ip_config_object['name']
-        orig_ip_address = ip_config_object['privateIpAddress']
+        orig_ip_address = ip_config_object['privateIPAddress']
         # Dynamic | Static
         orig_ip_allocation_method = ip_config_object['privateIpAllocationMethod']
```
Comment (haagha) [inline src/vm-repair/azext_vm_repair/custom.py:585]: Can you also update this? to  ip_config_object['privateIPAllocationMethod']

**Review by haagha — CHANGES_REQUESTED** [review]
Can you also update this:         orig_ip_allocation_method = ip_config_object['privateIpAllocationMethod']

File: src/vm-repair/azext_vm_repair/custom.py (hunk 1 above)
Comment (pselugar) [reply src/vm-repair/azext_vm_repair/custom.py]: Good catch, updated

**Review by haagha — APPROVED** [review]
Looks good!

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**